core/app/models/spree/stock/quantifier.rb

```diff
@@ -5,12 +5,13 @@ class Quantifier
 
       def initialize(variant)
         @variant = variant
-        @stock_items = Spree::StockItem.joins(:stock_location).where(variant_id: @variant, Spree::StockLocation.table_name => { active: true })
+        @stock_items = Spree::StockItem.joins(:stock_location).
+                         where(variant_id: @variant, Spree::StockLocation.table_name => { active: true })
```

Comment:
@aishmita-aggarwal - We already have an active scope in stock_location, I guess we should use that with merge here.

Any thoughts?